lib/remote/apilistener.ti

```diff
@@ -49,22 +49,7 @@ class ApiListener : ConfigObject
 
 	[config] String ticket_salt;
 
-	[config] Array::Ptr access_control_allow_origin {
-		default {{{ return new Array(); }}}
-	};
-	[config] bool access_control_allow_credentials
-	{
-		default {{{ return true; }}}
-	};
-	[config] String access_control_allow_headers
-	{
-		default {{{ return "Authorization"; }}}
-	};
-	[config] String access_control_allow_methods
-	{
-		default {{{ return "GET, POST, PUT, DELETE"; }}}
-	};
-
```

Comment:
If these attributes are removed, they should at least being marked as deprecated and kept without functionality in 2.9. Otherwise upgrades immediately break.

Docs aren't updated yet too.